web/package/agama-web-ui.changes

```diff
@@ -1,7 +1,13 @@
+-------------------------------------------------------------------
+Tue Sep 2 08:24:38 UTC 2025 - David Diaz <dgonzalez@suse.com>
```

Comment:
I see you were cleaning surplus spaces. Great!
But I was actually expecting two spaces between "Sep" and "2" (I think that's the default in this format for days with just one digit). But I don't think that really matters.

Reply:
Originally I keep it. In fact, I added it because I solved the conflict manually using the Github web interface... but CI complained about wrong date and the only thing I found the extra space. I can add it back, though.